core/drivers/crypto/caam/caam_desc.c

```diff
@@ -69,37 +69,40 @@ void caam_desc_add_ptr(uint32_t *desc, paddr_t ptr)
 }
 
 #ifdef CFG_CAAM_64BIT
-void caam_desc_push(uint64_t *in_entry, paddr_t paddr)
+/*
+ * Functions [put/get]_[le/be]64() handle 32bits words operations, meaning that
+ * words can be 4 bytes aligned.
```

Comment:
"can be" or "are"?

Reply:
I mean "can be". Is something wrong with this sentence?

Reply:
I think they **are always** at least 4 bytes aligned, which is what put_*/get_*() functions expect if i'm right.

This inline description suggests (at least to me, and i guess it is not of importance) that these _words may_ also be not 4byte aligned.
I opinion is that you should better simply remove this inline comment.

By the way, I suggest to use `if (IS_ENABLED(xxx)` in the 64bit variants of these functions.

Reply:
"are at least 4 bytes aligned" might be better indeed.
Anyway, I can remove this comment if you think it is useless.

About the ```IS_ENABLED``` macro, I know we now prefer this to ```#ifdef ... #endif``` but the CAAM driver has been built with ```#ifdef ...#endif``` everywhere.
If I use ```IS_ENABLED``` here, I should propagate the change to all CAAM files then for consistency. I rather do it in another PR.

Reply:
Reading again the sentence... and ok, I understand want you meant.
Yet, I still think it is a comment that could be useful in the commit message, but not really useful in the source code. Whatever, do as you wish.  
`Acked-by: Etienne Carriere <etienne.carriere@linaro.org>`

Reply:
@clementfaure 
> If I use `IS_ENABLED` here, I should propagate the change to all CAAM files then for consistency. I rather do it in another PR.

No big deal in any case, but I agree with you.